_overviews/scala3-book/scala-for-python-devs.md

```diff
@@ -68,17 +68,16 @@ This section looks at the similarities you’ll see between Python and Scala whe
 - The syntax for defining methods is similar
 - Both have lists, dictionaries (maps), sets, and tuples
 - Both have comprehensions for mapping and filtering
-- Both have higher-order functions and strong support for lambdas
-- With Scala 3’s toplevel definitions you can put method, field, and other definitions anywhere
-  - One difference is that Python can operate without even declaring a single method, while Scala 3 can’t do _everything_ at the toplevel; for instance, a `@main def` method is required to start a Scala application
+- With Scala 3’s [toplevel definitions][toplevel] you can put method, field, and other definitions anywhere
+  - One difference is that Python can operate without even declaring a single method, while Scala 3 can’t do _everything_ at the toplevel; for instance, a [main method][main-method] (`@main def`) is required to start a Scala application
 
 ### Programming level differences
 
 Also at a programming level, these are some of the differences you’ll see every day when writing code:  
 
 - Scala’s syntax is extremely consistent:
```

Comment:
Here you say "syntax", but then the most points that follow talk about ecosystem (library, style of coding, etc.)